examples/preprocessing/plot_interpolate_bad_channels.py

```diff
@@ -31,13 +33,20 @@
 
 fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
 evoked = mne.read_evokeds(fname, condition='Left Auditory',
-                          baseline=(None, 0))
+                          baseline=(None, 0), proj=False)
```

Comment:
I should revert this as it has no effect for these data, will do with whatever changes @agramfort or others request...